nginx.json

```diff
@@ -0,0 +1,44 @@
+{
+	"nginx": {
+		"description": "Open source reverse proxy server for HTTP, HTTPS, SMTP, POP3, and IMAP protocols, as well as a load balancer, HTTP cache, and a web server.",
+		"version": "1.0.0",
+		"website": "https://www.nginx.com/",
+		"icon": "https://cdn-1.wp.nginx.com/wp-content/themes/nginx-theme/assets/img/logo.svg",
+		"more_info": "You must <a href='https://gist.github.com/JasonTarka/63d632b5633fb0840cdfb3b56a14d388'>create a default config file</a> in the Config Storage share before starting this Rockon, which you can extend with your own custom configuration.<br/>For advanced users, an example configuration file can be found <a href='https://www.nginx.com/resources/wiki/start/topics/examples/full/'>in the nginx wiki</a>. The config volume is mounted at <pre>/etc/nginx/conf.d/</pre>",
+		"containers": {
+			"nginx": {
+				"image": "nginx",
+				"tag": "latest",
+				"launch_order": 1,
+				"ports": {
+					"80": {
+						"description": "HTTP port",
+						"host_default": 1080,
+						"label": "HTTP port",
+						"protocol": "tcp",
+						"ui": true
+					},
+					"443": {
+						"description": "HTTPS port.",
+						"host_default": 1443,
+						"label": "HTTPS port",
+						"protocol": "tcp",
+						"ui": true
+					}
+				},
+				"volumes": {
+					"/etc/nginx/conf.d": {
+						"description": "Choose a Share for the nginx configuration. Eg: create a Share called nginx-config for this purpose alone.<br/>You must <a href='https://gist.github.com/JasonTarka/63d632b5633fb0840cdfb3b56a14d388'>add a default config file</a>.",
```

Comment:
```suggestion
						"description": "Choose a Share for the nginx configuration. Eg: create a Share called nginx-config for this purpose alone.",
```

Reply:
I've moved the link for your example config from the share description to the general description as links are not processed in the tooltip used against each share.